docs/tutorials/index.rst

```diff
@@ -10,3 +10,4 @@ Tutorials
    get-started
    get-started-helm
    driving-flux
+   get-started-kustomize
```

Comment:
Would move this below `get-started-helm`

Reply:
Done!